index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+    <div class=" boxContainer">
+        <div class="boxNav">
+            <img src="images/My Logo.jpg" alt="My Logo">
+        </div>
+        <div>
+            <img class="boxMenu" src="images/Union.png" alt="Menu">
+        </div>
+    </div>
+    </header>
+    <div class="boxHeader">
+        <h1 class="boxHead">
+            I’m Abd elhakim
+            Glad to see you!
+        </h1>
+        <p class="boxPar">
+            I’m a software developer! I can help you<br> build a product ,
+             feature or website Look<br> through some of my work and<br> experience!
+              If you like what you see and<br> have a project you need coded,
+               don’t<br> hesitate to contact me.
+        </p>
+        <p class="boxConnect">
+            LET’S CONNECT
+        </p>
+        <div class="boxSocial">
+            <img src="images/social media icons.png">
+        </div>
```

Comment:
- [ ] Dear @abdozayan12, for your social media icons, We need to have them as individual Icons as a list item, The list of icons should be wrapped in a `ul` tag to group them, and `li` per icon; each icon having a link (`a` tag).  Apart from being semantically correct to arrange icons this way, the `a` tag will let you connect your social networks with the icons, using href.
> something like this
```html
<ul class="">
       <li>
          <a href=""><img src="./images/"alt="twitter"></a>
        </li>
         <li>
          <a href="#"><img src="./images/"alt="linkedln"></a>
          </li>
        <li>
          <a href="#"><img src="./images/"alt="media icon"></a>
        </li>
        <li>
          <a href="#"><img src="./images/"alt="github"></a>
        </li>
        <li>
          <a href="#"><img src="./images/"alt="thumbsup"></a>
        </li>

      </ul>
```